### DISCUSSION TITLE
[Help] Cannot upload (most) assets to the server

### DISCUSSION THREAD

**Discussion by gregoryduckworth** [immich-app/immich#2473] [Q&A]
So I've a reasonably standard setup:
- Latest Immich 1.56 docker setup (although this problem has been existing before then)
- pfSense + HAProxy infront

The situation is when I upload most files they typically always fail and in the console of the browser I get:
<img width="1003" alt="image" src="https://github.com/immich-app/immich/assets/2647926/39c0ccd5-3709-4d77-a937-c23b8849943e">

It works for very few assets but I can get some to upload:
<img width="1212" alt="image" src="https://github.com/immich-app/immich/assets/2647926/21112a97-5b89-4b0e-a612-671a697ee995">

As the requests are failing on the browser side, there are no logs coming through to the docker containers to be able to provide them.

Is anyone experiencing similar issues or might have any idea on what might be causing the problem?

**Comment by alextran1502** [comment]
Do you have size limitation of your proxy?
